tests/test_result_to_dict.py

```diff
@@ -200,6 +203,16 @@ def test_callback(self):
         data = result_to_dict(json_value_without_str_obj, callback=convert_return_none)
         self.assertDictEqual(data, json_value_without_str_obj)
 
-    @ResultToDict()
-    def get_from_params(self, param):
-        return param
+    def test_pymongo_to_dict(self):
+        with connect_to_mongo().get() as client:
+            collection = client.testing_db.example
+            data_not_converted = collection.find()
+            data_converted = result_to_dict(collection.find())
+            self.assertEqual(type(data_converted), type([1, 2, 3]))

```

Comment:
use `assertIsInstance` and `assertNotIsInstance` methods here to check the equality of type